### ISSUE TITLE
Login screen - poor image quality

### ISSUE THREAD

**Issue by bbusenius** [hahnicity/MEGAndroid#19]
The logo text that says, "MEG Mobile Encryption Gateway" is poor quality. It would be best if this could be setup as an .svg.

**Comment by JoshuaLyle** [comment]
Just to confirm, you're talking about the main screen that leads to logging in, qr management, and registration correct?

**Comment by bbusenius** [comment]
Yes, your are correct, though image quality is poor throughout. I don't remember if I opened a separate bug for the icons on the registration page but those need to be replaced too.
